gef.py

```diff
@@ -3609,7 +3607,7 @@ def keystone_assemble(code, arch, mode, *args, **kwargs):
         ks = keystone.Ks(arch, mode)
         enc, cnt = ks.asm(code, addr)
     except keystone.KsError as e:
-        err("Keystone assembler error: {:s}".format(str(e)))
+        err(f"Keystone assembler error: {str(e):s}")
```

Comment:
redundant: `:s` already calls `str` on it. Maybe grep for this pattern elsewhere.